vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/ProjectInformationDialog.java

```diff
@@ -97,21 +112,43 @@ private void initForm() {
     formLayout.add(searchField);
     formLayout.add(titleField);
     formLayout.add(projectObjective);
-    formLayout.add(experimentalDesignField);
+    formLayout.add(experimentalDesignIntroduction);
+    formLayout.add(sampleCountLayout);
+    formLayout.add(organismBox);
+    formLayout.add(specimenBox);
+    formLayout.add(analyteBox);
     formLayout.add(projectManager);
     formLayout.add(principalInvestigator);
-    // set form layout to only have one column (for any width)
+    // Set FormLayout with one column
     formLayout.setResponsiveSteps(new ResponsiveStep("0", 1));
   }
 
   private void styleSearchBox() {
     searchField.setPlaceholder("Search");
     searchField.setClassName("searchbox");
-    searchField.addClassNames("flex",
-        "flex-row",
-        "w-full",
-        "min-width-300px",
-        "max-width-15vw");
+    searchField.setMaxWidth(50, Unit.VW);
+    searchField.setMinWidth(50, Unit.VW);
+  }
+
+  private void styleSampleCountField() {
+    sampleCountField.setMinWidth(150, Unit.PIXELS);
+    sampleCountField.setValue(1);
+    sampleCountField.setMin(1);
+    sampleCountField.setStep(1);
+    sampleCountField.setStepButtonsVisible(true);
+    sampleCountField.addThemeVariants(TextFieldVariant.LUMO_ALIGN_CENTER);
+  }
+
+  private void initExperimentalDesignIntroduction() {
+    Span experimentalDesignHeader = new Span("Experimental Design");
+    Span experimentalDesignDescription = new Span(
+        "Describe the experimental design by the following fields. Multiple values are allowed");
```

Comment:
`Describe the experimental design` might be misleading. This is not what you do here. You provide some basic information for the experimental design which (probably?) is described later?

Reply:
I updated the description to specify more clearly that it's the sample information that's requested here 👍